docs/src/main/paradox/openshift.md

```diff
@@ -0,0 +1,99 @@
+# Deploy on OpenShift
+
+### Prerequisites
+
+Install the following:
+
+* [Docker](https://docs.docker.com/install/)
+* [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+* OpenShift's CLI: [`oc`](https://docs.openshift.com/container-platform/3.10/cli_reference/get_started_cli.html#installing-the-cli)
+* [Sbt](https://www.scala-sbt.org/)
+
+
+#### Preface
+
+There are [multiple flavors](https://www.openshift.com/products?extIdCarryOver=true&sc_cid=701f2000001OH7iAAG) of `oc` and OpenShift. This guide was tested with:
+
+```
+$ oc version
+
+oc v3.10.45
+kubernetes v1.10.0+b81c8f8
+features: Basic-Auth
+
+Server https://centralpark.lightbend.com:443
```

Comment:
I think we should replace each mention to centralpark.lightbend.com by <your-openshift-server>